src/server/index.jsx

```diff
@@ -112,7 +110,7 @@ server
  * Application env routes
  */
 server
-  .get([articleSwPath, homePageSwPath], (req, res) => {
+  .get([homePageSwPath], (req, res) => {
```

Comment:
Now that there is only 1 route, we can remove the `[]` (both a string and a string array are supported parameters

```suggestion
  .get(homePageSwPath, (req, res) => {
```